docs/index.css

```diff
@@ -151,6 +151,11 @@ h6:has(> .Permalink):hover .Permalink,
   margin: var(--space-large);
 }
 
+.SearchField .TextFieldWrapper .page {
+  flex-shrink: 0;
+  margin-right: var(--space-smallest);
+}
+
```

Comment:
We've largely moved away from using global documentation styles here, you can see [Panel](https://cauldron.dequelabs.com/components/Panel) as an example. You could include a style property in the documentation to help demonstrate that the component does not come with inherited styles.

I'm also thinking `flex-shrink` may need to be applied at the component level since we likely don't want components in the wrapper to wrap:

```css
.TextFieldWrapper > :not(input) {
  flex-shrink: 0;
}
```